docs/HelpStyleGuide.md

```diff
@@ -0,0 +1,66 @@
+# Style Guide for User-Facing Help Text in LORIS
+
+Help text exists as an on-screen guide for users, to add context and provide instructions to using all of the features in LORIS. It is always accessed by clicking the question mark in the top right menu bar in LORIS, as seen below:
+
+![Help Text](/docs/images/helptext.png)
+
+## How to Store and Name Files
+
+Help text content is stored into the aces/Loris repo as markdown (.md) files within the help/subdirectory of the module it belongs to.. For example, the Help text for Imaging Browser main page is located here: `modules/imaging_browser/help/imaging_browser.md`. 
+
+Some modules have more than one Help text file to correspond with subpages. These files exist in the same help/ folder, but are named differently - they must match exactly the php/file name that spawns the page. For example, the Imaging Browser module has a subpage called “ViewSession”. Its help file is stored as: `modules/imaging_browser/help/viewsession.md`. 
+
+## Markdown
+
+Formatting should follow markdown guidelines, with some exceptions. This is because LORIS’s current markdown template may not cover all possible markdown features (i.e. bullets). Help text should not include HTML tags for security reasons. When creating a markdown file, use only the following styles, or else talk to the LORIS technical team about expanding our Markdown features:
+
+* `*Italics*`
+* `**Bold**`
+* `***Italics and Bold***`
+* `# Heading 1` up to `###### Heading 6`
+  * Use `#H1` for the name of the module at the beginning of the help text
+  * Use `#H2` for any subheadings
+  * Use `#H4` for sub-subheadings
+* Avoid using HTML line break (`<br>`) tags—simply use physical line breaks (spaces between lines) where necessary. 
+  * Using `<br>` might be necessary only in lists of items
+  
+  ## When to Use Emphasis
+  
+Whether a user knows it or not, they will come to recognize styles that are consistent throughout the text. This way, their eyes can jump to a part of the text that addresses their issue, if they don’t need to read the whole thing. Follow these rules on using emphasis:
+
+* *Italicize* when referring to a specific place on the screen
+  * i.e. in the Selection Filters section
+  * i.e. in the Metadata column of the table
+  * i.e. in the Imaging menu
+* **Bold** when referring to any clickable button, including modules and tabs
+  * DON’T call it a button
+    * i.e. Click **Submit** (rather than “click the **Submit** button”)
+  * DO call it a module or tab
+    * i.e. In the **Imaging Browser** module
+    * i.e. Click the **Upload** tab
+  * You can also bold when defining features in a list, to separate the term being defined from the definition itself, i.e. when describing columns in a table:
+     **Column heading 1**: description
```

Comment:
These aren't rendering on their own lines.  Take a look, @knoll-alex ?

Reply:
thanks! this should be fixed.